src/vnsw/agent/vrouter/ksync/ksync_flow_memory.cc

```diff
@@ -126,9 +141,8 @@ void KSyncFlowMemory::InitFlowMem() {
     }
     nl_free_client(cl);
 
+#if defined(__linux__)
```

Comment:
I suggest you to use here ifdef like in the rest of the file

Reply:
Original file uses "#if defined" version (see lines: 6, 8).